schema/appdal/application.schema.xml

```diff
@@ -331,8 +331,8 @@
 
  <class name="SmartDaqApplication" is-abstract="yes">
   <superclass name="Application"/>
-  <relationship name="queue_rules" class-type="QueueConnectionRule" low-cc="zero" high-cc="many" is-composite="no" is-exclusive="no" is-dependent="no"/>
-  <relationship name="network_rules" class-type="NetworkConnectionRule" low-cc="zero" high-cc="many" is-composite="no" is-exclusive="no" is-dependent="no"/>
+  <relationship name="queue_rules" class-type="QueueConnectionRule" low-cc="one" high-cc="many" is-composite="no" is-exclusive="no" is-dependent="no"/>
```

Comment:
I think it is quite conceivable that a SmartDaqApplication has only one DAQModule and therefore no Queues so why force low-cc to one rather than zero?

Reply:
I have to admit that I don't remember making that change, and assumed it's because of using git rebase rather than git merge in the past. Definitely agree with you here. Should I change it back to zero together with the `network_rules` below, or only for the `queue_rules`?

Reply:
Yes, ahh, I see you've done it already!